tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -345,7 +345,7 @@ function adjust_qemu_cmdline() {
 	# The paths on the kata-containers tarball side look like:
 	# ${dest_dir}/opt/kata/share/kata-qemu/qemu
 	# ${dest_dir}/opt/kata/share/kata-qemu-snp-experimnental/qemu
-	[[ "${shim}" =~ ^(qemu-snp|qemu-nvidia-snp)$ ]] && qemu_share=${shim}-experimental
+	[[ "${shim}" =~ ^(qemu-nvidia-gpu-snp|qemu-nvidia-gpu-tdx)$ ]] && qemu_share=${shim}-experimental
```

Comment:
I actually think that what you had before was correct:
```sh
ls -l opt/kata/share/ | grep -E 'snp|tdx'
drwxr-xr-x    3 root     root          4096 May 14 09:43 kata-qemu-snp-experimental
drwxr-xr-x    3 root     root          4096 May 14 09:43 kata-qemu-tdx-experimental
```

So, at this point the names are just:
qemu-snp and qemu-tdx

Reply:
I am confused ... `qemu-snp` needs to go since we use the upstream QEMU for CPU only workloads.
Now the shim is not called `qemu-nvidia-snp` it is `qemu-nvidia-gpu-snp` and `qemu-nvidia-gpu-tdx` respectively... Need to check.

Reply:
Hmm. I think you're right.